extensions/arc/src/ui/dashboards/postgres/postgresOverviewPage.ts

```diff
@@ -74,29 +73,19 @@ export class PostgresOverviewPage extends DashboardPage {
 			CSSStyles: titleCSS
 		}).component());
 
-		this.kibanaLink = this.modelView.modelBuilder.hyperlink()
-			.withProperties<azdata.HyperlinkComponentProperties>({
-				label: this.getKibanaLink(),
-				url: this.getKibanaLink()
-			}).component();
+		this.kibanaLink = this.modelView.modelBuilder.hyperlink().component();
 
-		this.grafanaLink = this.modelView.modelBuilder.hyperlink()
-			.withProperties<azdata.HyperlinkComponentProperties>({
-				label: this.getGrafanaLink(),
-				url: this.getGrafanaLink()
-			}).component();
+		this.grafanaLink = this.modelView.modelBuilder.hyperlink().component();
 
 		this.kibanaLoading = this.modelView.modelBuilder.loadingComponent()
 			.withItem(this.kibanaLink)
-			.withProperties<azdata.LoadingComponentProperties>({
-				loading: !this._controllerModel.endpointsLastUpdated
-			}).component();
+			.component();
 
 		this.grafanaLoading = this.modelView.modelBuilder.loadingComponent()
 			.withItem(this.grafanaLink)
-			.withProperties<azdata.LoadingComponentProperties>({
-				loading: !this._controllerModel.endpointsLastUpdated
-			}).component();
+			.component();
+
+		this.refreshDashboardLinks();
```

Comment:
Not sure about this change.  It was done the old way to work around bug https://github.com/microsoft/azuredatastudio/issues/11141.  I had to set the component's value before adding it to the loading component.  It's also why the sql page does it in this order:

```
// Update loaded components with data
// Assign the loading component after it has data
```

Perhaps the bug doesn't reproduce anymore.  I haven't checked recently.

Reply:
I'll play around with it, but I tried loading the page a few times and refreshing it and didn't notice anything weird.

Reply:
@bergeron Updated with the same logic as MIAA - I did indeed see issues with opening an already-refreshed dashboard but moving to that way of doing things solved that.